html/browsers/browsing-the-web/navigating-across-documents/about-srcdoc-navigation-blocked.window.js

```diff
@@ -0,0 +1,43 @@
+// META: title=Navigation to about:srcdoc, not via srcdoc="", must be blocked
+// META: script=../resources/helpers.js
```

Comment:
Yes, I tried to stack this on top of the other PR, but it looks like GitHub is not great about rebasing if you change the base PR (which I did in response to your review there).

In general don't worry about reviewing this yet. It might need some expansion or tweaks when I get back to fixing the javascript: URL spec rewrite. I'll move it to draft.